src/Rules.Framework/RulesEngine.cs

```diff
@@ -23,6 +24,7 @@ namespace Rules.Framework
     /// </typeparam>
     public class RulesEngine<TContentType, TConditionType> : IRulesEngine<TContentType, TConditionType>
     {
+        private static readonly RuleValidator<TContentType, TConditionType> ruleValidator = new();
```

Comment:
Since it's being used in 2 different classes: RulesEngine and RuleBuilder, I changed the RuleValidator a little bit, modify its constructor to private, and added a static method to return an Instance for both classes. Let me know what you think.
![image](https://user-images.githubusercontent.com/1099060/217095122-169baeaa-6fae-4199-ad0c-52a14a643d91.png)

Reply:
In other words, you've implemented the Singleton pattern, right? 😝 cool! 👌🏼 🙂